ecosystem/Alternate-backends.md

```diff
@@ -6,6 +6,7 @@ There are various alternatives to `psc`'s default JavaScript backend:
 |:--------------------------------------------------------------------------------------|:-----------|:--------------|:------|:-----------------------|
 | [andyarvanitis/purescript-native](https://github.com/andyarvanitis/purescript-native)             | 0.13.x    | C++11 or Go         | active | all applicable tests in `purescript/tests/purs/passing` are passing |
 | [purerl/purerl](https://github.com/purerl/purerl) | 0.13.5 | Erlang | active |
+| [purenix-org/purenix](https://github.com/purenix-org/purenix) | | Nix | active | |
```

Comment:
```suggestion
| [purenix-org/purenix](https://github.com/purenix-org/purenix) | 0.14.4 | Nix | active | |
```

This looks good to me!  Thanks for opening a PR to add this.

cc @jonascarpay

Reply:
Not sure if people were waiting for me to sign off on this, but yes, thanks, LGTM!